e2e/pages/citizen/createCase/initialJourney/caseDashboardPage.ts

```diff
@@ -85,4 +91,16 @@ export class CaseDashboardPage {
       childArrangementsJourney as keyof typeof linkSelectors;
     await page.click(linkSelectors[journeyTypeKey]);
   }
+
+  public static async selectDraftCase({
+    page,
+    accessibilityTest,
+    ccdRef,
+  }: SelectDraftCaseOptions): Promise<void> {
+    await this.checkPageLoads({
+      page,
+      accessibilityTest,
+    });
+    await page.getByRole("link", { name: `${ccdRef}` }).click(); // Select draft case
```

Comment:
```suggestion
    await page.getByRole("link", { name: ccdRef }).click(); // Select draft case
```

Don't need the string interpolation here